libs/sdk-core/src/lsp.rs

```diff
@@ -56,7 +57,10 @@ impl LspInformation {
                 .into_iter()
                 .map(|ofp| ofp.into())
                 .collect::<Vec<OpeningFeeParams>>(),
-        }
+        };
+
+        OpeningFeeParamsMenu::try_from(info.opening_fee_params_menu.clone())?;
```

Comment:
It looks a bit weird that we first convert the opening_fee_params to our own structure and then validate. I is more natural flow to validate the raw input and then convert it.
If this is the only point we need to validate then maybe move the validate to this struct implementation?

Reply:
Addressed in the latest commit:

https://github.com/breez/breez-sdk/pull/292/commits/8e3d706a1eea7a6a728d4514cea8620ca23086ce#diff-209db7ea9780093f9c880649e7f40a90a4615da7cee3111a299bbfa430b3aec2L55-L62

Part of it was also:
- changing `LspInformation.opening_fee_params_menu` type to `OpeningFeeParamsMenu`, so the menu is validated on instantiation
- moved `OpeningFeeParamsMenu` to `models.rs`
- updated uniffi bindings